charts/devlake/Chart.yaml

```diff
@@ -29,7 +29,7 @@ type: application
 
 
 # Chart version
-version: 0.15.0-rc5
+version: v0.15.1-beta3
 
 # devlake version
-appVersion: "v0.15.0-rc5"
+appVersion: "vv0.15.1-beta3"
```

Comment:
why not refer by variable since the same tag has been defined in `values.yaml:131`?

Reply:
I tried, but failed...